instance/models/instance.py

```diff
@@ -86,7 +87,8 @@ class Instance(ValidateModelMixin, TimeStampedModel):
     STARTED = 'started'
     ACTIVE = 'active'
     BOOTED = 'booted'
-    PROVISIONED = 'provisioned'
+    PROVISIONING = 'provisioning'
```

Comment:
@chromano Will the field value be properly migrated for existing instances, if one happen to be in the `provisioned` state when this is deployed?